src/ProjectService.Broker/Requests/Interfaces/IDepartmentService.cs

```diff
@@ -13,5 +14,7 @@ public interface IDepartmentService
       List<string> errors,
       List<Guid> departmentsIds = null,
       List<Guid> usersIds = null);
+
+    Task<DepartmentUserRole?> CheckDepartmentUserRoleAsync(Guid departmentId, Guid userId, List<string> errors = null);
```

Comment:
Not sure if "check" is suitable since you're getting it

Reply:
You're right, "get" is better